src/features/Integrations/IntegrationFormField.tsx

```diff
@@ -114,11 +129,38 @@ const FieldContent: FC<{
           </Markdown>
         </Box>
       );
+    case "string":
+      if (f_size === "array") {
+        return (
+          <ArgumentsTable
+            initialData={values ? (values[fieldKey] as string[]) : []}
```

Comment:
i'd avoid type casting here, see if adding a type check inf the `if` statement will work :)